src/apps/service/apps/agave/jobs.clj

```diff
@@ -119,6 +119,7 @@
     :batch           false
     :description     (:description job)
     :enddate         (:enddate job)
+    :system_id       "agave"
```

Comment:
Pretty minor and I only thought of it because I just reviewed the mescal PR, but might it make sense to pull this from `mescal.agave-de-v2.constants/hpc-system-id`? Seems like an easily-overlooked lone place it might need changing, if for some reason it needs changing.